### ISSUE TITLE
Generating code using @ngrx/schematics is really slow

### ISSUE THREAD

**Issue by arutkowski00** [ngrx/platform#1102]
## Minimal reproduction of the bug/regression with instructions:
1. Create app with ngrx/store via Angular CLI
2. Go to any directory
3. Generate some code using ngrx/schematics:
```
ng g feature foo
ng g effect bar
```

## Expected behavior:
Code should be generated as fast as from Angular schematics, instead I have to wait around 30-40 seconds for anything to be outputted.

## Versions of NgRx, Angular, Node, affected browser(s) and operating system(s):
Angular: 6.0.3
NgRx: 6.0.1
Node: 8.11.1
OS: Windows 7 Enterprise SP1 x64

## I would be willing to submit a PR to fix this issue

[ ] Yes (Assistance is provided if you need help submitting a pull request)
[x] No

**Comment by brandonroberts** [comment]
There are some things we can do to speed this up. Instead of calling functions whenever we need to define a camel case/uppercase/snake case/etc name, we could define these and just use them in the schematics templates.

**Comment by tdeschryver** [comment]
I'm fairly certain that this is because we're filtering paths in the schematics of effect, reducer and store.

```
filter(path => path.endsWith('.module.ts') && !path.endsWith('-routing.module.ts')),
```

This line will check every file in your code base, including the node_modules.

What I don't know is, why we're doing this 😅. 
To try it out I removed those checks and the tests are all passing, and I tried some commands in an empty project and they seem alright.
